net/sched/sch_skbprio.c

```diff
@@ -134,15 +120,13 @@ static int skbprio_enqueue(struct sk_buff *skb, struct Qdisc *sch,
 
 	q->qstats[lp].backlog -= qdisc_pkt_len(to_drop);
 	q->qstats[lp].drops++;
-
-	__skb_queue_tail(qdisc, skb);
-	qdisc_qstats_backlog_inc(sch, skb);
-	q->qstats[prio].backlog += qdisc_pkt_len(skb);
+	q->qstats[lp].overlimits++;
 
 	/* Check to update highest and lowest priorities. */
 	if (skb_queue_empty(lp_qdisc)) {
 		if (q->lowest_prio == q->highest_prio) {
-			BUG_ON(sch->q.qlen);
+
+			/* The incoming packet is the only packet in queue. */
```

Comment:
Drop the blank line above the comment.

One last improvement, after the comment add the line:
`BUG_ON(sch->q.qlen == 1);`

Reply:
Fixing my typo above: `BUG_ON(sch->q.qlen != 1);`